src/App.js

```diff
@@ -19,9 +20,12 @@ function App() {
           <Route path="/products" >
             <ProductsPage />
           </Route>
-          <Route path="/">
+          <Route exact path="/">
```

Comment:
in this case random url strings will send us to a blank page, you could remove exact and switch with your /clock path, which would still show your /clock page, but in all other cases show home page